### ISSUE TITLE
[UX] Edit view from Layout blocks page

### ISSUE THREAD

**Issue by laryn** [backdrop/backdrop-issues#3501]
**Issue**
It's frustrating when I'm on the **Layout > Manage Blocks** page and working with a block that is provided by a View, and then need to tweak that View but have no way to get to it:

![screen shot 2019-01-25 at 2 16 08 pm](https://user-images.githubusercontent.com/871421/51770635-0e6d0400-20ac-11e9-8b33-827ef3eca87e.jpg)

So I have to go to **Structure > Views**, find that View, load it up, then find the relevant block display within that View... (or go back to a page on the front end that the View block shows up on and use the Contextual links to edit the View).

**Suggestion**
It would be very nice to have an "Edit View" link in the drop list, that jumps directly to the relevant block display within that View automatically.

**Comment by klonos** [comment]
I agree that this would be a very handy UX improvement 👍 (another put-things-where-people-expect-to-find-them thing); although I would prefer it if the link was somewhere in the block description, rather as a dropbutton item. I think that dropbutton links should include only items that that act on the block itself - not whatever it is that generates/provides it.
